core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
I'm complaining ;)

Why here? Are we going to do that for *everything*? You can easily make Cromwell explode by doing dumb stuff in a config. What's special about drawing the line here on this one?

I really don't like the idea of putting these values in extremely non-standard places just because someone might be a dumbass and override stuff that isn't documented as something they should be playing with.